pkg/controllers/ctrlutil/work.go

```diff
@@ -39,16 +39,14 @@ import (
 // CreateOrUpdateWork creates a Work object if not exist, or updates if it already exists.
 func CreateOrUpdateWork(ctx context.Context, c client.Client, workMeta metav1.ObjectMeta, resource *unstructured.Unstructured, options ...WorkOption) error {
 	resource = resource.DeepCopy()
-	if workMeta.Labels[util.PropagationInstruction] != util.PropagationInstructionSuppressed {
```

Comment:
I can't remember why skip setting labels for the Work with `propagation.karmada.io/instruction:suppressed` label.
@XiShanYongYe-Chang Do you know why?

Reply:
The skip operation should be set only for resources collected from member clusters. These labels are meaningless.